script.js

```diff
@@ -9,4 +9,483 @@ function closeMenu() {
+divVerticalLines3.appendChild(divVerticalLine33);
+
+paragraphVerticalLines43.textContent = 'Ruby';
+divVerticalLines3.appendChild(paragraphVerticalLines43);
+
+buttonWorkSection13.textContent = 'See Project';
+buttonWorkSection13.setAttribute('type', 'button');
+buttonWorkSection13.setAttribute('onclick', 'topFunction()');
+buttonWorkSection13.classList.add('btn-transition', 'openWindow3');
+section13.appendChild(buttonWorkSection13);
+
+// Work Section 4
+const divWorkSpace114 = document.createElement('div');
+const divWhiteScreenLeft24 = document.createElement('div');
+const section114 = document.createElement('section');
+const header114 = document.createElement('h2');
+const paragraph114 = document.createElement('p');
+const divVerticalLines24 = document.createElement('div');
+const paragraphVerticalLines114 = document.createElement('p');
+const divVerticalLine114 = document.createElement('div');
+const paragraphVerticalLines224 = document.createElement('p');
+const divVerticalLine224 = document.createElement('div');
+const paragraphVerticalLines324 = document.createElement('p');
+const paragraphVerticalLines424 = document.createElement('p');
+const divVerticalLine324 = document.createElement('div');
+const buttonWorkSection124 = document.createElement('button');
+
+divWorkSpace114.classList.add('work-space-2');
+works.appendChild(divWorkSpace114);
+
+divWhiteScreenLeft24.classList.add('white-screen-right');
+divWorkSpace114.appendChild(divWhiteScreenLeft24);
+
+section114.classList.add('work-section-description');
+divWorkSpace114.appendChild(section114);
+
+header114.textContent = `${projects[0].name}`;
+section114.appendChild(header114);
+
+paragraph114.classList.add('work-p');
+paragraph114.textContent = `${projects[0].descriptionMobile}`;
+section114.appendChild(paragraph114);
+
+divVerticalLines24.classList.add('vertical-lines', 'flex');
+section114.appendChild(divVerticalLines24);
+
+paragraphVerticalLines114.textContent = 'css';
+divVerticalLines24.appendChild(paragraphVerticalLines114);
+
+divVerticalLine114.classList.add('vertical-line');
+divVerticalLines24.appendChild(divVerticalLine114);
+
+paragraphVerticalLines224.textContent = 'html';
+divVerticalLines24.appendChild(paragraphVerticalLines224);
+
+divVerticalLine224.classList.add('vertical-line');
+divVerticalLines24.appendChild(divVerticalLine224);
+
+paragraphVerticalLines324.textContent = 'bootstrap';
+divVerticalLines24.appendChild(paragraphVerticalLines324);
+
+divVerticalLine324.classList.add('vertical-line');
+divVerticalLines24.appendChild(divVerticalLine324);
+
+paragraphVerticalLines424.textContent = 'Ruby';
+divVerticalLines24.appendChild(paragraphVerticalLines424);
+
+buttonWorkSection124.textContent = 'See Project';
+buttonWorkSection124.setAttribute('type', 'button');
+buttonWorkSection124.setAttribute('onclick', 'topFunction()');
+buttonWorkSection124.classList.add('btn-transition', 'openWindow4');
+section114.appendChild(buttonWorkSection124);
```

Comment:
- [ ] **[Optional] You are repeating the code for creating a project for each project. This will make it very difficult to manage the code. This can be improved**

One of the things that make your code clean is the DRY rule. DRY stands for don't repeat yourself. You should not repeat the same code again and again. Suppose you had to make some changes for every project, you will have to go through the code for every project you are creating. This will just be tiresome and prone to error.

You already have an array of projects. This array will contain all your projects. You can create a function to render the project. Then, you can loop through the array and call that function for each element.  This is one of the ways you can reduce code.